shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -135,6 +143,16 @@
     public static final String STORAGE_METADATA_TABLE_GET_LATENCY = PREFIX + "segmentstore.storage.metadata_table_get_latency_ms";      // Histogram
     public static final String STORAGE_METADATA_TABLE_WRITE_LATENCY = PREFIX + "segmentstore.storage.metadata_table_write_latency_ms";  // Histogram
 
+    public static final String STORAGE_METADATA_FOUND_IN_TXN = PREFIX + "segmentstore.storage.metadata_found_in_txn";       // Counter
+    public static final String STORAGE_METADATA_FOUND_IN_BUFFER = PREFIX + "segmentstore.storage.metadata_found_in_buffer"; // Counter
+    public static final String STORAGE_METADATA_FOUND_IN_CACHE = PREFIX + "segmentstore.storage.metadata_found_in_cache";   // Counter
+    public static final String STORAGE_METADATA_FOUND_IN_STORE = PREFIX + "segmentstore.storage.metadata_found_in_store";   // Counter
+    public static final String STORAGE_METADATA_NOT_FOUND = PREFIX + "segmentstore.storage.metadata_not_found";             // Counter
+
+    public static final String STORAGE_METADATA_BUFFER_SIZE = PREFIX + "segmentstore.storage.metadata_buffer_size";         // Gauge
```

Comment:
Always include the unit of measure for sizes. In this case, maybe add a `_bytes` suffix to all these sizes.

Reply:
changed names.